specs/Spec.Agile.HPKE.fst

```diff
@@ -104,11 +106,14 @@ let id_of_mode m =
 /// Constants sizes
 
 inline_for_extraction
-let size_aead_nonce (cs:ciphersuite): size_nat = AEAD.size_nonce (aead_of_cs cs)
+let size_aead_nonce (cs:ciphersuite): size_nat = 12
```

Comment:
It'd be nice to document where this value come from.
This could be in the form of a lemma that proves that this is the nonce length for all supported AEAD algorithms.